core/ballerina/dictionary_repeating_tags.bal

```diff
@@ -0,0 +1,2 @@
",\"retired\":\"Retired\",\"keyword\":\"OverlayNumberOfTables\"},\"60xx0803\":{\"vr\":\"AT\",\"vm\":\"1-n\",\"name\":\"Overlay Code Table Location\",\"retired\":\"Retired\",\"keyword\":\"OverlayCodeTableLocation\"},\"60xx0804\":{\"vr\":\"US\",\"vm\":\"1\",\"name\":\"Overlay Bits For Code Word\",\"retired\":\"Retired\",\"keyword\":\"OverlayBitsForCodeWord\"},\"60xx1001\":{\"vr\":\"CS\",\"vm\":\"1\",\"name\":\"Overlay Activation Layer\",\"retired\":\"\",\"keyword\":\"OverlayActivationLayer\"},\"60xx1100\":{\"vr\":\"US\",\"vm\":\"1\",\"name\":\"Overlay Descriptor - Gray\",\"retired\":\"Retired\",\"keyword\":\"OverlayDescriptorGray\"},\"60xx1101\":{\"vr\":\"US\",\"vm\":\"1\",\"name\":\"Overlay Descriptor - Red\",\"retired\":\"Retired\",\"keyword\":\"OverlayDescriptorRed\"},\"60xx1102\":{\"vr\":\"US\",\"vm\":\"1\",\"name\":\"Overlay Descriptor - Green\",\"retired\":\"Retired\",\"keyword\":\"OverlayDescriptorGreen\"},\"60xx1103\":{\"vr\":\"US\",\"vm\":\"1\",\"name\":\"Overlay Descriptor - Blue\",\"retired\":\"Retired\",\"keyword\":\"OverlayDescriptorBlue\"},\"60xx1200\":{\"vr\":\"US\",\"vm\":\"1-n\",\"name\":\"Overlays - Gray\",\"retired\":\"Retired\",\"keyword\":\"OverlaysGray\"},\"60xx1201\":{\"vr\":\"US\",\"vm\":\"1-n\",\"name\":\"Overlays - Red\",\"retired\":\"Retired\",\"keyword\":\"OverlaysRed\"},\"60xx1202\":{\"vr\":\"US\",\"vm\":\"1-n\",\"name\":\"Overlays - Green\",\"retired\":\"Retired\",\"keyword\":\"OverlaysGreen\"},\"60xx1203\":{\"vr\":\"US\",\"vm\":\"1-n\",\"name\":\"Overlays - Blue\",\"retired\":\"Retired\",\"keyword\":\"OverlaysBlue\"},\"60xx1301\":{\"vr\":\"IS\",\"vm\":\"1\",\"name\":\"ROI Area\",\"retired\":\"\",\"keyword\":\"ROIArea\"},\"60xx1302\":{\"vr\":\"DS\",\"vm\":\"1\",\"name\":\"ROI Mean\",\"retired\":\"\",\"keyword\":\"ROIMean\"},\"60xx1303\":{\"vr\":\"DS\",\"vm\":\"1\",\"name\":\"ROI Standard Deviation\",\"retired\":\"\",\"keyword\":\"ROIStandardDeviation\"},\"60xx1500\":{\"vr\":\"LO\",\"vm\":\"1\",\"name\":\"Overlay Label\",\"retired\":\"\",\"keyword\":\"OverlayLabel\"},\"60xx3000\":{\"vr\":\"OB or OW\",\"vm\":\"1\",\"name\":\"Overlay Data\",\"retired\":\"\",\"keyword\":\"OverlayData\"},\"60xx4000\":{\"vr\":\"LT\",\"vm\":\"1\",\"name\":\"Overlay Comments\",\"retired\":\"Retired\",\"keyword\":\"OverlayComments\"},\"7Fxx0010\":{\"vr\":\"OB or OW\",\"vm\":\"1\",\"name\":\"Variable Pixel Data\",\"retired\":\"Retired\",\"keyword\":\"VariablePixelData\"},\"7Fxx0011\":{\"vr\":\"US\",\"vm\":\"1\",\"name\":\"Variable Next Data Group\",\"retired\":\"Retired\",\"keyword\":\"VariableNextDataGroup\"},\"7Fxx0020\":{\"vr\":\"OW\",\"vm\":\"1\",\"name\":\"Variable Coefficients SDVN\",\"retired\":\"Retired\",\"keyword\":\"VariableCoefficientsSDVN\"},\"7Fxx0030\":{\"vr\":\"OW\",\"vm\":\"1\",\"name\":\"Variable Coefficients SDHN\",\"retired\":\"Retired\",\"keyword\":\"VariableCoefficientsSDHN\"},\"7Fxx0040\":{\"vr\":\"OW\",\"vm\":\"1\",\"name\":\"Variable Coefficients SDDN\",\"retired\":\"Retired\",\"keyword\":\"VariableCoefficientsSDDN\"}}";
```

Comment:
Is this something customizable by the user? If so we have to consider on providing ability to configure

Reply:
No. Repeating group tags are part of standard DICOM tags, so these shouldn’t be user customizable.